straxen/get_corrections.py

```diff
@@ -119,10 +103,9 @@ def get_correction_from_cmt(run_id, conf):
         return conf
     if isinstance(conf, tuple) and len(conf) == 3:
         is_nt = conf[-1]
```

Comment:
also for `get_correction_from_cmt` now would be a good time to add a docstring what inputs are expected and what the function does

Reply:
done